src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -367,12 +367,14 @@ def setupProcessingTask(self, procTask, taskType, inputDataset=None, inputStep=N
         procTaskCmsswHelper.setOverrideCatalog(self.overrideCatalog)
         procTaskCmsswHelper.setErrorDestinationStep(stepName=procTaskLogArch.name())
 
-        if forceMerged:
+        if forceMerged and isinstance(forceMerged, bool):
             procTaskStageHelper.setMinMergeSize(0, 0)
-        elif forceUnmerged:
+        elif forceUnmerged and isinstance(forceUnmerged, bool):
```

Comment:
It seems `forceMerged` and `forceUnmerged` are supposed to be either `False` or a list. Which makes me think we should not check for boolean type here. 

Instead, I'd keep the simple checks and default these args to None in the method signature. The same for `def addOutputModule`. The code will look more readable IMO.